### ISSUE TITLE
UDim2 invalid member errors

### ISSUE THREAD

**Issue by SadisticNub** [Anaminus/roblox-bug-tracker#801]
Calling an invalid member on a UDim2 throws an odd error message: "...  is not a valid member, did you forget to capitalize the first letter?"

Example:
> UDim2.new(0):asd()
- asd is not a valid member, did you forget to capitalize the first letter?

I have not found any other instances where this also applies.

**Comment by The0neThe0nly** [comment]
Seems like it is intended behavior to me.

**Comment by SadisticNub** [comment]
I don't think you understand what I meant - no other types give this error. It doesn't seem intentional to me.

**Comment by LucasWolschick** [comment]
Seems like a safeguard of sorts, still the message is pretty weird.
